apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -740,32 +765,38 @@ private void initEditText(int key, final int title, final String preferencesKey,
 
                 @Override
                 public boolean onPreferenceClick(final Preference preference) {
-                    final EditText input = new EditText(getActivity());
-                    input.setInputType(InputType.TYPE_CLASS_TEXT);
-                    input.setText(MainActivity.sSharedPreferences.getString(preferencesKey, defaultValue));
-
-                    new AlertDialog.Builder(getActivity()).setTitle(title).setView(input)
-                            .setPositiveButton(R.string.save, new DialogInterface.OnClickListener() {
-                                @Override
-                                public void onClick(DialogInterface dialog, int which) {
-                                    listener.onSave(input.getText().toString());
-                                    preference.setSummary(getSummary(preferencesKey, defaultValue));
-                                }
-                            })
-                            .setNeutralButton(R.string.reset, new DialogInterface.OnClickListener() {
-                                @Override
-                                public void onClick(DialogInterface dialog, int which) {
-                                    listener.onReset();
-                                    preference.setSummary(getSummary(preferencesKey, defaultValue));
-                                }
-                            })
-                            .setNegativeButton(R.string.cancel, null)
-                            .create().show();
-                    return true;
+                    return ShowPreferenceTextEditor(preference, title, preferencesKey, defaultValue, listener);
                 }
             });
         }
 
+        private boolean ShowPreferenceTextEditor(final Preference preference, final int title, final String preferencesKey, final String defaultValue, final EditTextListener listener)
```

Comment:
This is a C# method name, but here it's a Java file. In Java method names are camelCase.

Reply:
Thanks, fixed